src/reducers/move/__tests__/MoveReducer-test.js

```diff
@@ -0,0 +1,18 @@
+import { assert } from '../../../../test/setup'
+
+import { moveObject } from '../index'
+
+const sendToBackend = [
+  {
+    doneBy: 3,
+    items: [1],
+    destination: 1
+  }
+]
+
+
+describe('MoveReducer', () => {
+  it('Call move object', () => {
+    assert(sendToBackend !== moveObject(1, [1], 1))
```

Comment:
its for sure that sendToBackend does not equal the response of moveObject :) nice try but no free lunch. We can rework this tomorrow.